spec/tasks/whitehall_importer_spec.rb

```diff
@@ -118,6 +138,24 @@
     expect { importer.import }.to raise_error(Tasks::AbortImportError)
   end
 
+  it "raises AbortImportError when revision history is missing for state" do
+    import_data["editions"][0]["state"] = "published"
+    importer = Tasks::WhitehallImporter.new(123, import_data)
+
+    expect { importer.import }.to raise_error(Tasks::AbortImportError)
```

Comment:
It looks like this class isn't namespaced within WhitehallImporter. We should have it as `Tasks::WhitehallImporter::AbortImportError` - I must have missed that the first time this was setup. Otherwise there is nothing to tie this to `WhitehallImporter`